includes/polling/os/fabos.inc.php

```diff
@@ -1,4 +1,10 @@
 <?php
-
 $version  = trim(snmp_get($device, '1.3.6.1.4.1.1588.2.1.1.1.1.6.0', '-Ovq'), '"');
-$hardware = trim(snmp_get($device, 'ENTITY-MIB::entPhysicalDescr.1', '-Ovq'), '"');
+$gethardware = trim(snmp_get($device, 'SNMPv2-SMI::mib-2.75.1.1.4.1.3.1', '-Ovq'), '"');
+$revboard = str_replace("SNMPv2-SMI::enterprises.1588.2.1.1.", "", $gethardware);
```

Comment:
This just seems wrong :)

You can probably get what you want by using different -O flags, what does an snmpget look like for it?